### PR TITLE
Add flatbuffers to third_party to avoid issues with top-level build

### PR DESCRIPTION
**Pull request by rdspring1**
Fixes https://github.com/NVIDIA/Fuser/issues/137

### CODE REVIEW

File: FlatBuffers.cmake [hunk 1]
```diff
@@ -0,0 +1,11 @@
+set(FlatBuffers_Include ${PROJECT_SOURCE_DIR}/third_party/flatbuffers/include)
```
Comment (jjsjann123) [inline FlatBuffers.cmake:1]: nitpick, can we put cmake related things inside a folder? something like `./cmake`

File: csrc/python_frontend/fusion_record.h [hunk 2]
```diff
@@ -2727,8 +2727,6 @@ struct RandomOpRecord : RecordFunctor {
     for (auto& it : output_shape_) {
       fb_shape.push_back(serde::State(it.index, it.stype));
     }
-    auto shape_fb =
```
Comment (jjsjann123) [inline csrc/python_frontend/fusion_record.h:2730]: this seems to be just a clean up?

File: tools/update_copyright.py [hunk 3]
```diff
@@ -34,6 +34,7 @@
     "./version.txt",
     "./test/main.cpp",
     "Dependencies.cmake",
+    "FlatBuffers.cmake",
```
Comment (jjsjann123) [inline tools/update_copyright.py:37]: Thanks for doing this.

**Review by jjsjann123 — APPROVED** [review]
LGTM.

**Comment by jjsjann123** [comment]
!build

**Review by zasdfgbnm — APPROVED** [review]
Working on my system now. Thanks for the fix!

File: csrc/python_frontend/fusion_record.h (hunk 2 above)
Comment (rdspring1) [reply csrc/python_frontend/fusion_record.h:2730]: It causes an error with some compilers.

**Pull request merged**